### PR TITLE
Tidy up structure and release packages

### PR DESCRIPTION
**Pull request by pezholio**
I've tried to tidy up the structure a bit, and added the packages for each platform. These should work - would be good if people with Linux / Windows machines could test though :)

### CODE REVIEW

**Comment by pezholio** [comment]
(Also, I've undone @bcouston's visual changes - lovely as they are :wink: )

File: README.md [hunk 1]
```diff
@@ -4,11 +4,21 @@
...
+* [Linux 64 bit](https://raw.githubusercontent.com/theodi/comma-chameleon/master/packages/comma-chameleon-linux-x64.tar.gz)
+* [Windows 32 bit](https://raw.githubusercontent.com/theodi/comma-chameleon/master/packages/comma-chameleon-win32-ia32.tar.gz)
+* [Windows 64 bit](https://raw.githubusercontent.com/theodi/comma-chameleon/master/packages/comma-chameleon-win32-x64.tar.gz)
+
+## Development setup
 
 ```
 brew install node
-npm install bower jasmine electron-prebuilt -g
+cd app/
```
Comment (danielgavrilov) [inline README.md]: Don't we still need `bower` and `electron-prebuilt`?

Comment (pezholio) [inline README.md]: Yeah, good point!

**Comment by pezholio** [comment]
Don't merge this yet! I need to make some changes :smile:

**Comment by quadrophobiac** [comment]
@pezholio I've tried running this locally but I cannot get the Datapackage popup window to function - I get a blank popup window when I open it

**Comment by quadrophobiac** [comment]
Merging this, ticket opened to find less space intensive way of distributing the binaries

**Pull request merged**